src/Config.js

```diff
@@ -43,6 +44,16 @@ class ConfigOptions {
         for (let v of allOptions) {
             assert(v !== undefined );
         }
+
+        let developerPairs = this._coreDevelopers.split(',').map(i => i.trim());
+        this._coreDeveloperIds = new Map();
```

Comment:
If possible, I would reorder to better "announce" what we are after and make the temporary/helper list constant:

```suggestion
        this._coreDeveloperIds = new Map();
        const developerPairs = this._coreDevelopers.split(',').map(i => i.trim());
```

Reply:
Done.